airflow/contrib/operators/content_email_operator.py

```diff
@@ -0,0 +1,104 @@
+from airflow.models import BaseOperator
+from airflow.utils import send_MIME_email
+from airflow.utils import apply_defaults
+from airflow.hooks import PrestoHook
+from email.MIMEMultipart import MIMEMultipart
+from email.MIMEText import MIMEText
+from email.MIMEImage import MIMEImage
+import logging
+import jinja2
+import os
+import contrib.content_email_components as content_email_components
+import subprocess
+
+class ContentEmailOperator(BaseOperator):
+
+    template_fields = ('data','subject','variables')
+    template_ext = ('.sql')
+    ui_color = '#f0ede4'
+
+    @apply_defaults
+    def __init__(
+            self,
+            to,
+            subject,
+            data,
+            template,
+            variables,
+            presto_conn_id='presto_default',
```

Comment:
If this works only with Presto it should be named as such. I think it should be easy to generalize it to work with any hooks that has the required methods.